src/components/atoms/StoryCardML8x5DP7x5HeadlineOverlay/storyCardML8x5DP7x5HeadlineOverlay.m.css

```diff
@@ -1,7 +1,7 @@
 .base {
   --background-color: var(--white);
   display: grid;
-  grid-template-columns: 1fr 1fr;
+  grid-template-columns: 1fr 2fr;
```

Comment:
Mobile hero image seems to be having white space at the bottom due to change in template columns, is this expected?

Reply:
got the issue , its because of aspect ratio, have fixed it